src/main/java/seedu/multitasky/model/EntryBook.java

```diff
@@ -144,6 +144,11 @@ public void resetData(ReadOnlyEntryBook newData) {
      * and updates the Tag objects in the entry to point to those in {@link #tags}.
      */
     public void addEntry(ReadOnlyEntry e) throws DuplicateEntryException {
```

Comment:
@kevinLamKB 
This will possibly pass a `DuplicateEntryException` to `DeleteByFindCommand.java` and `DeleteByIndexCommand.java`. Need your help to fix those red boxes in these two files.